### ISSUE TITLE
null in query variables is wrongfully passed as a string "null"

### ISSUE THREAD

**Issue by bcgodev** [jensneuse/graphql-go-tools#286]
I am using v1.25.0 of `graphql-go-tools`. I found out that a `null` value in the variables will be passed by `graphql-go-tools` as a string `"null"`. It also happens in v1.23 too.

Using `NewFederationEngineConfigFactory` from 1.25.0, I am able to create a gateway

```
factory := graphql.NewFederationEngineConfigFactory
```

and using the `NewGraphqlHTTPHandler` in `examples/federation/gateway/http` to create a handler.

To showcase the problem, I put my mutation string and variables below

```
mutation ($city: String) {
  updatemember(id: "81", data: {city: $city}) {
    city
  }
}
```

```
{
	"city": null
}
```

However, at my graphql_datasource server, it recerives the following `JSON`. The city has a string value `"null"`.

```
{"variables":{"a":"81","city":"null"},"query":"mutation($a: ID!, $city: String){updatemember(id: $a, data: {city: $city}){city}}"}
```

It only happens when `null` is assigned in variables. If `null` is in the input, it would be passed as `null`.

It's expected that the variables follow the same behavior as stated in [graphql spec](https://github.com/graphql/graphql-spec/blob/main/spec/Section%202%20--%20Language.md)

```
Null Value

NullValue : null

Null values are represented as the keyword {null}.
```

Or did I use the tool in a wrong way? Any help is appreciated. Thanks in advance.

**Comment by jensneuse** [comment]
I think this should be solved by this PR, can you try?
https://github.com/TykTechnologies/graphql-go-tools/pull/145

**Comment by bcgodev** [comment]
> I think this should be solved by this PR, can you try?
> [TykTechnologies#145](https://github.com/TykTechnologies/graphql-go-tools/pull/145)

Thanks for the quick replay. I can gladly confirm that it does fix it.
<img width="795" alt="Screen Shot 2021-09-16 at 00 16 14" src="https://user-images.githubusercontent.com/21715194/133470805-83091792-66b6-41d6-88b6-9161c88e2758.png">

**Comment by jensneuse** [comment]
Should now be fixed with 1.26.0.
Can you confirm (and close) or report additional problems.

**Comment by bcgodev** [comment]
Tested with 1.26.1 and it's fixed.
